README.md

```diff
@@ -57,7 +57,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: O(1) and O(1) nested - are both constants.  O(1)
```

Comment:
This is actually O(n * m). Because we are iterating through two arrays, each in linear time, so we say O(n * m) to indicate that the two iterated arrays aren't the same.